app/controllers/administrative-units/administrative-unit/related-organizations/edit.js

```diff
@@ -14,100 +14,13 @@ export default class AdministrativeUnitsAdministrativeUnitRelatedOrganizationsEd
   @service store;
 
   get hasValidationErrors() {
-    return this.model.administrativeUnit.isInvalid;
+    return this.model.administrativeUnit.error;
   }
 
   queryParams = ['sort'];
 
   @tracked sort = 'name';
 
-  get isWorshipAdministrativeUnit() {
-    return this.isWorshipService || this.isCentralWorshipService;
-  }
-
-  get isProvince() {
-    return (
-      this.model.administrativeUnit.classification?.get('id') ===
-      CLASSIFICATION_CODE.PROVINCE
-    );
-  }
-
-  get isOCMW() {
-    return (
-      this.model.administrativeUnit.classification?.get('id') ===
-      CLASSIFICATION_CODE.OCMW
-    );
-  }
-
-  get isMunicipality() {
-    return (
-      this.model.administrativeUnit.classification?.get('id') ===
-      CLASSIFICATION_CODE.MUNICIPALITY
-    );
-  }
-
-  get isAgb() {
-    return (
-      this.model.administrativeUnit.classification?.get('id') ===
-      CLASSIFICATION_CODE.AGB
-    );
-  }
-
-  get isApb() {
-    return (
-      this.model.administrativeUnit.classification?.get('id') ===
-      CLASSIFICATION_CODE.APB
-    );
-  }
-
-  get isDistrict() {
-    return (
-      this.model.administrativeUnit.classification?.get('id') ===
-      CLASSIFICATION_CODE.DISTRICT
-    );
-  }
-
-  get isIgs() {
-    return (
-      this.model.administrativeUnit.classification?.get('id') ===
-        CLASSIFICATION_CODE.PROJECTVERENIGING ||
-      this.model.administrativeUnit.classification?.get('id') ===
-        CLASSIFICATION_CODE.DIENSTVERLENENDE_VERENIGING ||
-      this.model.administrativeUnit.classification?.get('id') ===
-        CLASSIFICATION_CODE.OPDRACHTHOUDENDE_VERENIGING ||
-      this.model.administrativeUnit.classification?.get('id') ===
-        CLASSIFICATION_CODE.OPDRACHTHOUDENDE_VERENIGING_MET_PRIVATE_DEELNAME
-    );
-  }
-
-  get isPoliceZone() {
-    return (
-      this.model.administrativeUnit.classification?.get('id') ===
-      CLASSIFICATION_CODE.POLICE_ZONE
-    );
-  }
-
-  get isAssistanceZone() {
-    return (
-      this.model.administrativeUnit.classification?.get('id') ===
-      CLASSIFICATION_CODE.ASSISTANCE_ZONE
-    );
-  }
-
-  get isWorshipService() {
-    return (
-      this.model.administrativeUnit.classification?.get('id') ===
-      CLASSIFICATION_CODE.WORSHIP_SERVICE
-    );
-  }
-
-  get isCentralWorshipService() {
-    return (
-      this.model.administrativeUnit.classification?.get('id') ===
-      CLASSIFICATION_CODE.CENTRAL_WORSHIP_SERVICE
-    );
-  }
-
   get hasCentralWorshipService() {
```

Comment:
Imo this getter should also move to the model. It is duplicated in the [controller](https://github.com/lblod/frontend-organization-portal/blob/a2dbad5012c5d3aa0c5e2dfaa261588a3821b91a/app/controllers/administrative-units/new.js#L30) for new administrative units. (A similar argument can be made for moving (some) of the other remaining getters to the classification constants, but that is out of scope for this PR.)

Reply:
I hesitated, then I follow the same logic as the POC. But I agree it should be migrated to. I propose to migrate all remaining duplicate getters in controllers in a follow-up PR, between the form validation migration and the Ember update.
@mirdono What do you think?

Reply:
Dealing with the remaining duplicate getters, including the one I pointed out here, in a separate PR is fine for me. So no further objections for this PR.